reflex/compiler/compiler.py

```diff
@@ -189,18 +194,89 @@ def _compile_root_stylesheet(stylesheets: list[str]) -> str:
+
+                if not stylesheet_full_path.exists():
+                    raise FileNotFoundError(
+                        f"The stylesheet file {stylesheet_full_path} does not exist."
+                    )
+                elif not stylesheet_full_path.is_file():
+                    if stylesheet_full_path.is_dir():
+                        # NOTE: this can create an infinite loop, for example:
+                        # assets/
+                        #   | dir_a/
+                        #   |   | dir_c/ (symlink to "assets/dir_a")
+                        #   | dir_b/
+                        # so to avoid the infinite loop, we don't include symbolic links
+                        stylesheets += [
+                            str(p.relative_to(assets_app_path))
+                            for p in stylesheet_full_path.iterdir()
+                            if not (p.is_symlink() and p.is_dir())
+                        ]
+                        continue
+                    else:
+                        raise FileNotFoundError(
+                            f'The stylesheet path "{stylesheet_full_path}" is not a valid path.'
+                        )
+                elif (
+                    stylesheet_full_path.suffix[1:]
+                    not in constants.Reflex.STYLESHEETS_SUPPORTED
+                ):
+                    raise FileNotFoundError(
+                        f'The stylesheet file "{stylesheet_full_path}" is not a valid file.'
+                    )
+
+                if (
+                    stylesheet_full_path.suffix[1:]
+                    in constants.Reflex.STYLESHEETS_SUPPORTED
+                ):
+                    target = (
+                        Path.cwd()
+                        / constants.Dirs.WEB
+                        / constants.Dirs.STYLES
+                        / RE_SASS_SCSS_EXT.sub(".css", str(stylesheet)).strip("/")
+                    )
+                    target.parent.mkdir(parents=True, exist_ok=True)
+
+                    if stylesheet_full_path.suffix == ".css":
+                        target.write_text(
+                            data=stylesheet_full_path.read_text(),
+                            encoding="utf8",
+                        )
+                    else:
+                        if sass_compile is None:
+                            from sass import compile as sass_compile
+                        else:
+                            pass
+
+                        target.write_text(
+                            data=sass_compile(
+                                filename=str(stylesheet_full_path),
+                                output_style="compressed",
+                            ),
+                            encoding="utf8",
+                        )
+                else:
+                    pass
+
+                stylesheet = (
+                    f"./{RE_SASS_SCSS_EXT.sub('.css', str(stylesheet)).strip('/')}"
```

Comment:
it doesn't make sense to de regex when you have the file path

Reply:
i just replace the suffix now by the ".css" extensions (without regex)